e2e/stack_test.go

```diff
@@ -139,6 +196,11 @@ func TestE2E(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, txBlock.Transactions(), 2)
 
+	// inspect L1 for deposit tx receipt
+	receipt, err := l1Client.Client.TransactionReceipt(ctx, depositTx.Hash())
+	require.NoError(t, err, "deposit tx receipt")
+	require.NotNil(t, receipt, "deposit tx receipt")
```

Comment:
Is it worth asserting any information on the deposit tx receipt? It might be nice to verify that the `TransactionDeposited` event was emitted

Reply:
This suggestion bore fruit. As it stood, the transaction was reverting. Event verification added here: 5253de9. Adjustment to MaxGas that fixed the revert in the prior commit.